node-streams.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const fs             = require('fs');
+
+const split          = require('split2');
+const miss           = require('mississippi');
+
+const join           = require('./common/join');
+const exit0          = require('./common/exit0');
+const exit1          = require('./common/exit1');
+const readFile       = require('./common/read-file-callback');
+
+const main = () => {
+  const path = join(process.argv[2]);
+  let results;
+
+  miss.pipe(
+    fs.createReadStream(path('index.txt')),
+    split(),
+    miss.parallel(123, (fileName, cb) => readFile(path(fileName), cb)),
+    miss.concat(data => { results = data; }),
+    err => {
+      if (err) { return exit1(err); }
```

Comment:
Please use `err != null` to avoid type coercion.

Since `exit1` terminates the process, `return` is unnecessary.